scripts/packages_exporter.py

```diff
@@ -144,6 +162,14 @@ async def get_sign_keys(self):
         endpoint = 'sign-keys/'
         return await self.make_request('GET', endpoint)
 
+    async def download_repodata(self, repodata_path, repodata_url):
+        file_links = await get_repodata_file_links(repodata_url)
+        for link in file_links:
+            file_name = os.path.basename(link)
+            if not link.endswith('/'):
```

Comment:
1. You can always add slash to the end of url and it doesn't break anything
2. Use `from urllib.parse import urljoin` for joining uris. As I remember `os.path.join` can return incorrect result in some cases.

Reply:
Not actual